config/routes.rb

```diff
@@ -1,9 +1,13 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root 'users#index'
+  
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show] 
+    resources :posts, only: [:index, :show, :new, :create] do 
+      resources :comments, only: [:new, :create]
+      resources :likes, only: [:create]
+    end
```

Comment:
#- [ ] I think it would be great if you fix your comments, likes  and posts routes to have index and show as well to let the user be able to see all the comments and likes in a specific post because I couldn't find any routes to it when I am tring to see all the comments so here you should add them to comments likes and posts 📌💯🔥✨✅

The same route here but method get 
![image](https://user-images.githubusercontent.com/58553711/214209438-2cc4aee9-a44b-4828-888b-df110d3100ab.png)